ruby/README.md

```diff
@@ -26,7 +31,8 @@ First, ensure that upb/ is checked out:
 
 Then install the required Ruby gems:
 
-    $ sudo gem install bundler rake rake-compiler rspec rubygems-tasks
+    $ sudo gem install bundler
+    $ bundle
 
 Then build the Gem:
```

Comment:
Can you add some more detail here about running the tests under JRuby? We're trying to make the READMEs for each language detailed enough so that people without familiarity with each language's build system and tools can at least run the tests and generate a release. Thanks!